.github/workflows/test.yml

```diff
@@ -9,20 +9,18 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby_version: ["2.3.8", "2.4.5", "2.5.3", "2.6.3", "2.7.2"]
+        ruby_version: ["2.3", "2.4", "2.5", "2.6", "2.7", "3.0", "3.1"]
         experimental: [false]
         include:
-          - ruby_version: "3.0"
+          - ruby_version: "3.2"
             experimental: true
           - ruby_version: "ruby-head"
             experimental: true
-          - ruby_version: "jruby-head"
-            experimental: true
```

Comment:
The jruby build wouldn't finish in my experience. I haven't investigated the cause, some sort of deadlock perhaps.

I can put this back if people feel strongly.